### DISCUSSION TITLE
botocore latest version doesn't have the file data/emr/2009-03-31/service-2.json

### DISCUSSION THREAD

**Discussion by abhilash4942** [boto/botocore#3154] [Q&A] [answered]
I have upgraded my botocore version from 1.27.73 to 1.34.75 however the file "data/emr/2009-03-31/service-2.json" seems to be not available with new version and as as result we have few test cases failing while using "@moto.mock_emr"

Attached the image of the data/emr/209-03-31 folder whichi shows the files in it. 

<img width="310" alt="Screenshot 2024-04-04 164432" src="https://github.com/boto/botocore/assets/122959473/448d128c-c9be-45ee-8309-c9973aba22c6">

**Comment by mohdluqmancse** [comment]
Any updates on this?

**Comment by nateprewitt** [comment, marked as answer]
Hi @abhilash4942, @mohdluqmancse,

Thanks for reaching out and apologies for the delay. We can see that the `service-2.json` file is available in [1.34.75](https://github.com/boto/botocore/tree/1.34.75/botocore/data/emr/2009-03-31). We can also see it's present in your screenshot as `service-2.json.gz`.

The file is gzipped in our `.whl` distribution to reduce the size of the botocore artifact on disk. If you would like the uncompressed artifact, you can download the `.tar.gz` from PyPI with the `--no-binary` flag with `pip install`.

If there are issues with moto and the current wheels, I would reach out to the moto team. However, a quick search of their issue tracker suggests their issue was already idenfitied and addressed in November 2023 with getmoto/moto#7031. We would recommend starting with updating your version of moto.

**Comment by mohdluqmancse** [comment]
Thanks, I'm using moto==4.2.3 and this is fixed in 4.2.9 https://github.com/getmoto/moto/blob/master/CHANGELOG.md
